app/shared/components/BigButton.svelte

```diff
@@ -0,0 +1,64 @@
+<script type="ts" context="module">
+
+export function generateIcon(preferredJourneyMode:string){
+  if(preferredJourneyMode === "fastest") return "north"
+  if(preferredJourneyMode === "least-changes") return "switch_access_shortcut_add"
+  if(preferredJourneyMode === "least-walking") return "directions_walk"
+
+  return ""
+}
```

Comment:
This should not be part of a general BigButton Component. This should be included from outside